code/game/objects/structures/crates_lockers/closets/onestar.dm

```diff
@@ -0,0 +1,241 @@
+//Closets full of loot, they should be placed in derelicts
+
+//// Loot table on all tiers
+// Tier 1
+/obj/structure/closet/onestar/tier1
+	name = "\improper OneStar forgotten closet"
+	desc = "Old OneStar closet, doesn't look like it contains anything worthwhile. Maybe."
+	icon_state = "lootcloset"
+
+/obj/structure/closet/onestar/tier1/populate_contents()
+	new /obj/random/contraband/low_chance(src)
+	new /obj/random/contraband/low_chance(src)
+	new /obj/random/pack/rare/low_chance(src)
+	new /obj/random/junk(src)
+	new /obj/random/junk(src)
+	new /obj/random/junk(src)
+	new /obj/random/lowkeyrandom(src)
+	new /obj/random/lowkeyrandom(src)
+	new /obj/random/lowkeyrandom/low_chance(src)
+	new /obj/random/lowkeyrandom/low_chance(src)
+	new /obj/random/pack/tech_loot/low_chance(src)
+	new /obj/random/pack/cloth/low_chance(src)
+	new /obj/random/pack/cloth/low_chance(src)
+	new /obj/random/pack/gun_loot/low_chance(src)
+
+// Tier 2
+/obj/structure/closet/onestar/tier2
+	name = "\improper OneStar forgotten closet"
+	desc = "Old OneStar closet, it might have some good stuff inside it."
```

Comment:
"It's an old OneStar closet. Looks like there might be some decent stuff inside."